index.html

```diff
@@ -209,7 +209,11 @@ <h2>
         <dfn>Gamepad</dfn> interface
       </h2>
       <p>
-        This interface defines an individual gamepad device.
+        This interface represents an individual gamepad device. A Gamepad is a
+        live object; that is, user interaction with the device will be reflected
+        in the state of the object's properties. When the system receives new
+        data from a connected gamepad, a microtask is queued in the current
```

Comment:
Can probably drop "in the current event loop" as that's a bit ambiguous. 

I wonder if we should use the "user interaction task source"?

Reply:
That seems like the best approach. According to the HTML5 spec, input events should be queued this way:

"Asynchronous events sent in response to user input (e.g. click events) must be fired using tasks queued with the user interaction task source."

https://www.w3.org/TR/2014/REC-html5-20141028/webappapis.html#generic-task-sources

I think specifying the live object updates in terms of the user interaction task source will make it easier to spec gamepad button/axis events in the future.